src/main/java/frc/robot/subsystems/shooter/ShooterIOSim.java

```diff
@@ -35,7 +35,7 @@ public class ShooterIOSim implements ShooterIO {
       new DCMotorSim(DCMotor.getKrakenX60Foc(1), ShooterSubsystem.FLYWHEEL_RATIO, 0.001);
 
   ProfiledPIDController pivotController =
-      new ProfiledPIDController(1.0, 0.0, 1.0, new Constraints(10.0, 10.0));
+      new ProfiledPIDController(100.0, 0.0, 1.0, new Constraints(10.0, 10.0));
```

Comment:
Holy kP Batman! I know this is sim, but dang. Is this on purpose?

Reply:
it is on purpose. given that the sim doesnt have the same physical constraints, something which just gets to the target was desirable (prior behaviour was a very very slow final approach) this sim is mainly used in auto routines whose simulations are mainly for logic and drivetrain motion, not the specific physics of this mechanism. next season might want to pay more attention to setting sims like this up for timing validation, but given that this codebase is being retired its fine.